tests/first_argument_expansion/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -101,9 +101,12 @@ func(() => {
   thing();
 }, identifier);
 
-func(function() {
-  thing();
-}, this.props.timeout * 1000);
+func(
+  function() {
+    thing();
+  },
+  this.props.timeout * 1000
+);
```

Comment:
I _could_ change it from `isBinaryish()` to only `"LogicalExpression"`, but that seems like a compromise.